Tiltfile

```diff
@@ -343,6 +343,23 @@ def deploy_observability():
         k8s_yaml(read_file("./.tiltbuild/yaml/prometheus.observability.yaml"), allow_duplicates = True)
         k8s_resource(workload = "prometheus-server", new_name = "prometheus", port_forwards = "9090", extra_pod_selectors = [{"app": "prometheus"}], labels = ["observability"])
 
+    if "visualizer" in settings.get("deploy_observability", []):
+        internal_kubeconfig = str(local("kind get kubeconfig --name ${KIND_CLUSTER_NAME:-capi-test} --internal", quiet = True))
+        k8s_yaml(helm(
+            "./hack/observability/visualizer/charts/cluster-api-visualizer",
+            name = "visualizer",
+            namespace = "observability",
```

Comment:
>Build Failed: namespaces "observability" not found

I've been able to reproduce this and I'm not sure of the root cause, but removing the namespace here fixed this for me on my machine. Usual caveats apply 🤣

Reply:
It is probably deployed to default when you remove the namespace here (?)

Reply:
😄 Yeah - that's what's going on - let me take another look for a real solution.

Reply:
Please use the following instead:

```
    if "visualizer" in settings.get("deploy_observability", []):
        k8s_yaml(read_file("./.tiltbuild/yaml/visualizer.observability.yaml"), allow_duplicates = True)
        k8s_resource(
            workload = "capi-visualizer",
            new_name = "visualizer",
            port_forwards = [port_forward(local_port = 8000, container_port = 8081, name = "View visualization")],
            labels = ["observability"],
        )
```

tilt-prepare will render the helm chart + the namespce via the kustomization YAML automatically

Please use `namespace: {{ .Release.Namespace }}` in your Helm chart. Currently the visualizer always gets deployed in the `default` namespace